data/mods/CRT_EXPANSION/items/crt_ammo.json

```diff
@@ -23,7 +23,7 @@
     "id": "dhp_pellet",
     "copy-from": "pellet",
     "type": "AMMO",
-    "name": "domed HP pellets",
+    "name": { "str": "domed HP pellets" },
```

Comment:
You only applied the suggestion in my previous comment but did not update the other instances. All these "xxx_pellets" should use "str_sp" instead of "sp".

Reply:
"str_sp" instead of "sp".? I never did them all these mods are maintained and the mod author should be updating these, I'm only adding my additions, I have too many mods to update.

Reply:
"str_sp" instead of "str" I mean, sorry. You're changing names from strings to objects already so I think it would be better if you changed it to the most appropriate ones in this PR.